### PR TITLE
Update bundle to support new file format. Download files through api

### PR DESCRIPTION
**Pull request by rn0**
* [x] tag master as 2.0

### CODE REVIEW

**Comment by szymach** [comment]
master is already tagged as 2.0@dev

File: Teryt/Api/Client.php [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+        $resultKey = $functionName . 'Result';
+
+        return $this->prepareTempFile($response->{$resultKey}->plik_zawartosc);
+    }
+
+    private function makeCall($functionName, array $args)
+    {
+        $defaultSocketTimeout = ini_get('default_socket_timeout');
+        ini_set('default_socket_timeout', 30);
```
Comment (chives) [inline Teryt/Api/Client.php]: default value is 60

**Comment by rn0** [comment]
@JarekW

**Comment by JarekW** [comment]
dalej wip?

**Pull request merged**